libmaven/Compound.cpp

```diff
@@ -51,7 +51,6 @@ FragmentationMatchScore Compound::scoreCompoundHit(Fragment* f, float productPpm
 
         //theory fragmentation or library fragmentation = t
         //experimental data = f
-        t.sortByIntensity();
```

Comment:
Is this important? I could not tell what it was doing. With the new frag pair comparison method, re-sorts by m/z anyway.